index.js

```diff
@@ -33,6 +33,14 @@ module.exports = function inspect_ (obj, opts, depth, seen) {
     if (typeof obj === 'string') {
         return inspectString(obj);
     }
+    if (typeof obj === 'number') {
+      if (Object.is) {
```

Comment:
Surprisingly, `Object.is` is present as far back as node 0.8 :-). I've made this change so it works in 0.6.